module_14/tests/demoblaze_test.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from module_14.pages.home_page import HomePage
+from module_14.pages.product_page import ProductPage
+from module_14.pages.cart_page import CartPage
+
+
+@pytest.mark.usefixtures("setup")
+class TestDemoBlaze:
+
+    def test_login(self):
+        home_page = HomePage(self.driver)
+        home_page.open_home_page()
+        home_page.click_login_button()
+        home_page.log_in("snowfallslow@gmail.com", "1qa2ws3ed")
```

Comment:
It's better avoid hard-code like this and keep data as constants separately

Reply:
Thank you! Fixed

Reply:
Thank you for this point! Fixed.